src/_includes/macros/time.html

```diff
@@ -0,0 +1,44 @@
+{# ==========================================================================
+
+   time.render()
+
+   ==========================================================================
+
+   Description:
+
+   Renders a time element with a standard datetime format when given:
+
+   datetime: A date timestamp to format.
+
+   show_only (optional): Object of fields to show.
+                         Default is 'date', 'time', 'timezone'.
+
+   timezone (optional): Timezone string value. Default is 'America/New_York'.
+
+   ========================================================================== #}
+
+{% macro render(datetime,
+                show_only={'date':true, 'time':true, 'timezone':true},
+                timezone='America/New_York') %}
+{% from "macros/util/format/datetime.html" import format_datetime as format_datetime %}
+{% from "macros/util/format/datetime.html" import format_date as format_date %}
+{% from "macros/util/format/datetime.html" import format_time as format_time %}
+<time class="datetime" datetime="{{ format_datetime(datetime) }}">
+    {% if show_only.date == true  %}
+    <span class="datetime_date">{{ format_date(datetime) }}</span>
+    {% endif %}
+
+    {% if show_only.date == true and show_only.time == true %}
+    <span class="datetime_divider">@</span>
+    {% endif %}
+
+    {% if show_only.time == true %}
+    <span class="datetime_time">
+        {% if show_only.timezone == true %}
+        {{ format_time(datetime, timezone) }}</span>
+        {% else %}
+        {{ format_time(datetime) }}</span>
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/b10968cfc5a82eb26553fae59c30d1b218181314 Also, the divider shouldn't have been enclosed in the time element.